test.rb

```diff
@@ -53,23 +54,24 @@
 
 puts 'my_count'
 ary = [1, 2, 4, 2]
-puts ary.count               #=> 4
-puts ary.count(2)            #=> 2
-puts ary.count{ |x| x%2==0 } #=> 3
+puts ary.my_count               #=> 4
+puts ary.my_count(2)            #=> 2
+puts ary.my_count{ |x| x%2==0 } #=> 3
 puts "\n\n\n"
 
 puts 'my_map'
 puts (1..4).my_map { |i| i*i }      #=> [1, 4, 9, 16]
 puts "\n\n\n"
 
 puts 'my_inject'
-puts (5..10).my_inject { |sum, n| sum + n }         #=> 45
-puts (5..10).inject(1) { |product, n| product * n } #=> 151200
+puts (5..10).my_inject(2) { |sum, n| sum + n }         #=> 45
+puts (5..10).my_inject(2) { |product, n| product * n } #=> 151200
```

Comment:
**_#my_inject should raise a "LocalJumpError" when no block or argument is given_**